src/status_im/core.cljs

```diff
@@ -16,4 +17,5 @@
   (error-handler/register-exception-handler!)
   (status/init-jail)
   (.registerComponent react/app-registry "StatusIm" #(reagent/reactify-component app-root))
-  (re-frame/dispatch-sync [:initialize-app]))
+  (when @impl.module/jail-initialized?
```

Comment:
why do we need this here? if we have `:initialize-app` dispatch in jail handler?

Reply:
@flexsurfer good point, we don't actually need it here